apps/bare-expo/e2e/TestSuite-test.native.js

```diff
@@ -35,12 +35,16 @@ describe('test-suite', () => {
       `passes ${testName}`,
       async () => {
         const platform = device.getPlatform();
+        const launchArgs = {
+          EXDevMenuIsOnboardingFinished: true,
+        };
+        if (platform === 'android') {
+          launchArgs['class'] = 'dev.expo.payments.DetoxTest';
```

Comment:
now the `payments` and `detox` parts are both outdated in the application id 😂

Reply:
`payments` is true.

for detox, it is the class name: https://github.com/expo/expo/blob/d1f4129363af67f1525a636fc0ddcf42ab0f3726/apps/bare-expo/android/app/src/androidTest/java/dev/expo/payments/DetoxTest.java.
i still keep detox testing on android and think if someone needs to run it manually.
maybe to remove detox at some point like using maestro